tests/components/huisbaasje/test_sensor.py

```diff
@@ -0,0 +1,169 @@
+"""Test cases for the sensors of the Huisbaasje integration."""
+from homeassistant.components import huisbaasje
+from homeassistant.config_entries import CONN_CLASS_CLOUD_POLL, ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from tests.async_mock import patch
+
+MOCK_CURRENT_MEASUREMENTS = {
+    "electricity": {
+        "measurement": {
+            "time": "2020-11-18T15:17:24.000Z",
+            "rate": 1011.6666666666667,
+            "value": 0.0033333333333333335,
+            "costPerHour": 0.20233333333333337,
+            "counterValue": 409.17166666631937,
+        },
+        "thisDay": {"value": 3.296665869, "cost": 0.6593331738},
+        "thisWeek": {"value": 17.509996085, "cost": 3.5019992170000003},
+        "thisMonth": {"value": 103.28830788, "cost": 20.657661576000002},
+        "thisYear": {"value": 672.9781177300001, "cost": 134.595623546},
```

Comment:
Now that I have an idea of the Huisbaasje data-format, I would suggest you create sensors for thisWeek, thisMonth and thisYear too. You could disable these new sensors by default if you would want.

Reply:
How would that work, disabling those? Especially since all data is loaded in a single request

Reply:
Please see https://developers.home-assistant.io/docs/integration_quality_scale_index/#gold-%F0%9F%A5%87 the item before the last one for gold-quality.